src/lib/plugin.js

```diff
@@ -20,6 +20,8 @@ const util = require('util')
 const backoffMin = 1000
 const backoffMax = 30000
 const defaultConnectTimeout = 60000
+const wsReconnectDelayMin = 1
+const wsReconnectDelayMax = 300
```

Comment:
`reconnect()`'s `initialDelay` and `maxDelay` parameters should be in milliseconds -- are 1ms and 300ms the intended values?